packages/lodestar/src/opPool/modules/attestation.ts

```diff
@@ -17,6 +18,14 @@ export class AttestationOperations extends OperationsModule<Attestation> {
     this.config = config;
   }
 
+  public async verifyAndReceive(state: BeaconState, attestation: Attestation): Promise<boolean> {
```

Comment:
I was afraid we had the old `receive` method with only 1 single param, if I make it it would break things because of method overloading. But let me try.